examples/persistence_transform.py

```diff
@@ -0,0 +1,45 @@
+"""
+========================================
+Persistence Transform in Mapcubes
+========================================
+Persistence Transform is a simple image processing technique that is useful for the visualization
+and depiction of gradually evolving structures.
+This example illustrates persistence transform using the AIA_193_CUTOUT series.
+"""
+import matplotlib.pyplot as plt
+import sunpy.map
+import sunpy.physics.differential_rotation as diffrot
+from sunpy.data.sample import AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE
+import copy
+
+###############################################################################
+# We create the MapCube using the AIA_193_CUTOUT sample data.
+# To create a MapCube, we can call Map directly but add in a keyword to output a MapCube instead.
+aiamapcube = sunpy.map.Map(AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE,
+                           AIA_193_CUTOUT03_IMAGE, cube=True)
+
+###############################################################################
+# For a data set consisting of N images with intensity values I(x,y,t), the Persistence Map Pn is a
+# function of several arguments, namely intensity, location and time:
+# Pn(x,y,tn)=Q(I(x,y,t≤tn))
+maps_data = []
+persistence_maps = []
+persistence_maps.append(aiamapcube[0])
+maps_data.append(aiamapcube[0].data)
+for i in range(1, len(aiamapcube)):
+    value = aiamapcube[i].data/aiamapcube[i].data.max() < maps_data[i-1]
+    per = copy.copy(maps_data[i-1])
+    per[value] = aiamapcube[i].data[value]/aiamapcube[i].data.max()
+    smap = sunpy.map.Map(per, aiamapcube[i].meta)
+    maps_data.append(per)
+    persistence_maps.append(smap)
+
+###############################################################################
+# This plots the original mapcube
+aiamapcube.peek()
+
+###############################################################################
+# This plots the final mapcube after implementing the persistence transform
+result_mapcube = sunpy.map.MapCube(persistence_maps)
+result_mapcube.peek()
+plt.show()
```

Comment:
As they are, this returns blank images... With the changes I've proposed above then you can notice the persistance map.